GCTC-NTGC/gc-digital-talent
# Translations - Profiles

This branch provides a fresh found of French translations.

To test:
- Refresh your environment, including `npm run intl-compile`
- Browse the app in French and check for missing or broken translations
  - Ensure you're not using cached pages
  - Very recent changes are not yet translated, especially pool advertisement mutation messages like "Pool closed successfully!"
 
Closes #3531
